src/Plugin.php

```diff
@@ -55,21 +63,35 @@ public function activate(Composer $composer, IOInterface $io)
     }
 
     /**
-     * Install assets provided by the package, if any.
+     * Execute all installers.
+     */
+    public function onPostAutoloadDump()
+    {
+        while (0 < count($this->installers)) {
+            $installer = array_shift($this->installers);
+            $installer();
+        }
+    }
+
+    /**
+     * Memoize an installer for the package being installed.
      *
      * @param PackageEvent $event
      */
     public function onPostPackageInstall(PackageEvent $event)
     {
         $installer = new AssetInstaller($this->composer, $this->io);
-        $installer($event);
+        $this->installers[] = function () use ($event) {
+            $installer = new AssetInstaller($this->composer, $this->io);
+            $installer($event);
+        };
     }
 
     /**
      * Updates assets provided by the package, if any.
      *
      * Uninstalls any previously installed assets for the package, and then
-     * runs an install for the package.
+     * memoizes an install operation to run post-autoload-dump.
```

Comment:
typo in memorizes

Reply:
No typo: read about [memoization](https://en.wikipedia.org/wiki/Memoization) on wikipedia. :smile: